md/thermostat/utils.py

```diff
@@ -0,0 +1,19 @@
+import numpy as np
+from timemachine.constants import BOLTZ
+from simtk.unit import Quantity, kelvin
+
+
+def sample_velocities(masses: Quantity, temperature: Quantity) -> np.array:
+    """Sample Maxwell-Boltzmann velocities ~ N(0, sqrt(kB T / m)"""
+    n_particles = len(masses)
+    spatial_dim = 3
+
+    v_unscaled = np.random.randn(n_particles, spatial_dim)
+
+    # intended to be consistent with timemachine.integrator:langevin_coefficients
+    sigma = np.sqrt(BOLTZ * temperature.value_in_unit(kelvin)) * np.sqrt(1 / masses)
```

Comment:
I double checked this against https://github.com/choderalab/openmm-validation/blob/master/tests/test_custom_integrators.py#L62

and it seems consistent! nice.